CHANGELOG.md

```diff
@@ -6,7 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased](https://github.com/usgs/waterdataui/compare/waterdataui-0.19.0...master)
 ### Added
-- Added a second Y-axis for temperature parameters.
+- Added a second Y-axis for temperature parameters
+
+### Changed
+- Changed how the lower bound on symlog scales are calculated to be closer the minimum y-value on the plot
```

Comment:
nit: should it read "closer to the minimum"

Reply:
Yes. :flushed: